### PR TITLE
[SVCS-350] Adds endpoints to list renderer and exporter infomation

### PR DESCRIPTION
**Pull request by Johnetordoff**
## Purpose 

Currently there are not endpoints listing which renders and exporters are usable for MFR, this PR add endpoint which list that info.

## Changes

Adds to small handlers which respond with JSON dicts of available renders and exporters.

## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/browse/SVCS-350

### CODE REVIEW

File: tests/server/handlers/test_exporters.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+class TestExportersHandler(utils.HandlerTestCase):
+    @testing.gen_test
+    def test_get_status(self):
+        resp = yield self.http_client.fetch(self.get_url('/exporters'))
+
+        exporters = {}
+        for ep in pkg_resources.iter_entry_points(group='mfr.exporters'):
+            exporters.update({ep.name: ep.load().__name__})
+
+
```
Comment (cslzchen) [inline tests/server/handlers/test_exporters.py]: One empty line.

File: tests/server/handlers/test_renderers.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+class TestRenderersHandler(utils.HandlerTestCase):
+    @testing.gen_test
+    def test_get_status(self):
+        resp = yield self.http_client.fetch(self.get_url('/renderers'))
+
+        renderers = {}
+        for ep in pkg_resources.iter_entry_points(group='mfr.renderers'):
+            renderers.update({ep.name: ep.load().__name__})
+
+
```
Comment (cslzchen) [inline tests/server/handlers/test_renderers.py]: One empty line.

File: mfr/server/handlers/status.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import pkg_resources
```
Comment (cslzchen) [inline mfr/server/handlers/status.py]: This is where the test fails.

**Comment by cslzchen** [comment]
@Johnetordoff There is a unused import that needs to be fixed to pass the tests.

**Comment by felliott** [comment]
:+1:  Moving to RTM.

**Pull request closed without merging**